targets/targets.json

```diff
@@ -3504,7 +3504,7 @@
         }
     },
     "DISCO_L475VG_IOT01A": {
-        "components_add": ["QSPIF"],
+        "components_add": ["QSPIF", "FLASHIAP"],
```

Comment:
Hi
Did you check this new component with this DISCO_L475 target ?
I got all components-storage-blockdevice-component_flashiap-tests FAIL with 5.11.0 release... :-(

Reply:
Please create also a new issue for this

Reply:
Sorry about that. We have tested the block device with the general_test under feature/storage/TESTS/blockdevice and it uses get default instance and calculate the correct address and size. The component test fails because the base address and size are not set in the COMPONENT_FLASHIAP mbed_lib target overwrites. I run this with start address set to  0x801A800 and the size to 0xE5800 and it works. @jeromecoutant do you like me to add those to the mbed_lib.json permanently or you have some other numbers you prefer?

Reply:
Second thought that we will change the current implementation of the test and if no parameters will be provided in mbed_lib.app the test will calculate the start address and the size by itself as the get default instance does. Take a look at PR #9268